Use a regular string for cache-config-file option

nblock opened the pull request:
This option is loaded relative to conf/

treydock commented:
@nblock If you merge the updated master branch into your PR branch, the tests should pass.  I fixed the Ubuntu 20.04 tests.

nblock commented:
> @nblock If you merge the updated master branch into your PR branch, the tests should pass. I fixed the Ubuntu 20.04 tests.

rebased + force-pushed

treydock commented:
This will be released as v8.2.0 once Github Actions complete
